examples/struct.js

```diff
@@ -15,13 +15,13 @@ let xdr = XDR.config(xdr => {
 });
 
 let sig = new xdr.Signature();
-sig.publicKey(new Buffer(32));
+sig.publicKey(Buffer.alloc(32));
 sig.publicKey().fill(0x00);
```

Comment:
`Buffer.alloc` fills the buffer with `0`s by default so we don't need to call `fill`.